content/events/ethcc.md

```diff
@@ -1,5 +1,6 @@
 ---
 eventName: EthCC
+eventDescription: "See [who from the PLN](https://docs.google.com/spreadsheets/d/1mFeLwtTBBJiojxjD-mC27oupzOpY1xsy-po8axwiv4E/edit#gid=0) will be at EthCC and join [our Telegram group](https://t.me/+7BEhay59ceU1NTgx) for EthCC!"
```

Comment:
@laeccles There is support just for text content but not for md file formatting (such as hyperlinks).